### PR TITLE
map 19w07a resource reloading

### PR DESCRIPTION
**Pull request by asiekierka**
If this gets merged fast, I can push out API, so gogogo review!

### CODE REVIEW

File: mappings/com/mojang/blaze3d/platform/GlStateManager.mapping [hunk 1]
```diff
@@ -1,4 +1,6 @@
 CLASS com/mojang/blaze3d/platform/GlStateManager
+	CLASS com/mojang/blaze3d/platform/GlStateManager$DestFactor DestFactor
+	CLASS com/mojang/blaze3d/platform/GlStateManager$SourceFactor SourceFactor
```
Comment (JamiesWhiteShirt) [inline mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:3]: ?

Comment (asiekierka) [reply mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:3]: Matcher and Enigma disagree on inner class ordering :thinking:

Comment (UpcraftLP) [reply mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:3]: looks like he just sorted them, they were way below previously

Comment (asiekierka) [reply mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:3]: This is fine. Don't worry

**Review by modmuss50 — APPROVED** [review]
Without knowing the inner working of this I am not in a great position to provide better names, however I can look though it for obvious bad names.

I think this looks good, nothing is standing out to me. In this case names are better than no names. Nice work on figuring this out.

**Comment by asiekierka** [comment]
I have to thank Player for helping me clarify my mind. Anyhow, yeah, I'm merging this - we can improve this next time around.

**Pull request merged**